__tests__/components/editor/property/InputLang.test.js

```diff
@@ -29,33 +29,12 @@ describe('<InputLang />', () => {
     )
   })
 
-  it('typeahead component should use selectHintOnEnter', () => {
-    expect(wrapper.find('#langComponent').props().selectHintOnEnter).toBeTruthy()
+  it('typeahead component uses selectHintOnEnter', () => {
+    expect(wrapper.find('#langComponent').props().selectHintOnEnter).toEqual(true)
   })
 
-  it('should call the handleLangChange on change', () => {
+  it('calls the handleLangChange on change', () => {
     wrapper.find('#langComponent').simulate('change', [{ id: 'en', label: 'English' }])
-    expect(mockLangChangeFn.call.length).toBe(1)
-  })
-
-  it('should call the onFocus event and set the selected option', () => {
-    const opts = { id: 'URI', label: 'LABEL', uri: 'URI' }
-
-    wrapper.instance().opts = opts
-    const event = (wrap) => {
-      global.fetch = jest.fn().mockImplementation(async () => await { ok: true, resp: wrapper.instance().opts })
-      wrap.setState({ options: [wrapper.instance().opts] })
-      wrap.setState({ selected: [wrapper.instance().opts] })
-    }
-
-    wrapper.find('#langComponent').simulate('focus', event(wrapper))
```

Comment:
This test is not helpful.  here it calls the method defined on line 45, and then it checks that the method made changes to the state.   It is testing the test code itself, not the component under test.